src/ray/core_worker/core_worker_test.cc

```diff
@@ -1,19 +1,129 @@
+#include <thread>
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
 
 #include "core_worker.h"
+#include "context.h"
 #include "ray/common/buffer.h"
+#include "ray/raylet/raylet_client.h"
+
+#include <boost/asio.hpp>
+#include <boost/asio/error.hpp>
+#include <boost/bind.hpp>
+
+#include "ray/thirdparty/hiredis/async.h"
+#include "ray/thirdparty/hiredis/hiredis.h"
 
 namespace ray {
 
+std::string store_executable;
+std::string raylet_executable;
+
+ray::ObjectID RandomObjectID() { return ObjectID::from_random(); }
+
+static void flushall_redis(void) {
+  redisContext *context = redisConnect("127.0.0.1", 6379);
+  freeReplyObject(redisCommand(context, "FLUSHALL"));
+  freeReplyObject(redisCommand(context, "SET NumRedisShards 1"));
+  freeReplyObject(redisCommand(context, "LPUSH RedisShards 127.0.0.1:6380"));  
+  redisFree(context);
+}
+
 class CoreWorkerTest : public ::testing::Test {
  public:
-  CoreWorkerTest() : core_worker_(WorkerType::WORKER, Language::PYTHON) {}
+  CoreWorkerTest() {
```

Comment:
Maybe we should add `num_nodes` as a parameter? Because other tests may need only 1, or more than 2 nodes.

Reply:
makes sense. Thanks!